src/go/rpk/pkg/cli/cmd/wasm/deploy.go

```diff
@@ -69,6 +71,13 @@ func NewDeployCommand(
 	)
 	command.MarkFlagRequired("name")
 
+	command.Flags().StringVar(
```

Comment:
A string without error checking, the user could just pass in any value. Either do some error checking or modify the flag to be a boolean type with `--without-nodejs` and have that converted to the appropriate value before serializing

Reply:
I think `--without-nodejs` is not the best variant, because in future we will have `inline_wasm` type. The handler will be different for this type and data-policy

Reply:
Done